README.md

```diff
@@ -51,8 +53,22 @@ representation _(as described in RFC-4122)_.
 uuid(); // "52e6953d-edbe-4953-be2e-65ed3836b2f0"
 ```
 
-All random values in UUIDs produced by this API must be generated from a **[cryptographically
-secure][csprng]** source.
+## `Math.getRandomValues()`
+
+`Math.getRandomValues()` exposes an identical API to the
+[W3C `crypto.getRandomValues()`](https://www.w3.org/TR/WebCryptoAPI/#Crypto-method-getRandomValues)
+recommendation. With the same guarantees, regarding the quality of randomness:
+
+> Implementations should generate cryptographically random values using well-established
+> cryptographic pseudo-random number generators seeded with high-quality entropy, such as from an
+> operating-system entropy source (e.g., "/dev/urandom"). This specification provides no
+> lower-bound on the information theoretic entropy present in cryptographically random values, but
+> implementations should make a best effort to provide as much entropy as practicable.
+>
+> - [WebCryptoAPI 10.1. Description](https://www.w3.org/TR/WebCryptoAPI/#Crypto-description)
+
+`Math.getRandomValues()` will act as the foundation for implementing `UUID` algorithms, providing a
```

Comment:
NIT: we have so far not been writing all upper case `UUID` as code (which to me would suggest that to be literal part of the proposed api where we so far propose `uuid()`. Drop the quotes?